packages/@aws-cdk/aws-codedeploy/lib/trigger-configuration.ts

```diff
@@ -0,0 +1,20 @@
+import * as cdk  from '@aws-cdk/core';
+
+export interface TriggerConfiguration {
```

Comment:
Do `triggerName`s have to be unique? Because if so, an array is not really a great data structure for these.

Also, perhaps we could generate the names for these automatically, using construct IDs? Are these names visible in the console?